### PR TITLE
Readme/acknowledgments

### PR DESCRIPTION
**Pull request by AHReccese**
#### Reference Issues/PRs
- add PSF + Trelis badge in acknowledgement part
- update `CHANGELOG.md` which was missed in the previous PRs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +33,39 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 ### Changed
+- `ML Streaming` RESTful testcases
+- `attribute_call` function in `RESTServerCommunicator`
+- `AttributeCallPayload` class in `RESTServerCommunicator`
+- upload function in `RESTClientCommunicator`
+- download function in `RESTClientCommunicator`
+- `__init__` function in `RESTClientCommunicator`
+- `attribute_calls` function in `RESTClientCommunicator`
+- `requirements.txt`
+- `dev-requirements.txt`
```
Comment (sadrasabouri) [inline CHANGELOG.md:46]: specify additions to those files.

File: README.md [hunk 2]
```diff
@@ -177,6 +177,16 @@ You can also join our discord server
   <img src="https://img.shields.io/discord/1064533716615049236.svg?style=for-the-badge" alt="Discord Channel">
 </a>
 
+## Acknowledgments
+
+[Python Software Foundation (PSF)](https://www.python.org/psf/) grants PyMilo library partially for version **1.0**. [PSF](https://www.python.org/psf/) is the organization behind Python. Their mission is to promote, protect, and advance the Python programming language and to support and facilitate the growth of a diverse and international community of Python programmers.
+
+<a href="https://www.python.org/psf/"><img src="https://github.com/sepandhaghighi/pycm/raw/master/Otherfiles/PSF.png" height="55px" alt="Python Software Foundation"></a>
```
Comment (sadrasabouri) [inline README.md:184]: Are you sure you want to point out to `PSF.ong` in `sepandhaghighi/pycm`?
I would say it's better to add it to repository or point out to their website ([here](https://wiki.python.org/psf/PSF%20Press%20Kit)).

**Review by sadrasabouri — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (AHReccese) [reply README.md]: My bad! Thank you. 
I added the logo separately.

File: CHANGELOG.md (hunk 1 above)
Comment (AHReccese) [reply CHANGELOG.md]: sure, done.

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**